### PR TITLE
Add gqa parameter support to the server

### PR DESCRIPTION
**Pull request by IgnacioFDM**
Add gqa parameter support to the server
Also change help to stdout to follow https://github.com/ggerganov/llama.cpp/pull/2338

### CODE REVIEW

File: examples/server/server.cpp [hunk 1]
```diff
@@ -724,6 +725,13 @@ static void server_params_parse(int argc, char **argv, server_params &sparams,
             }
             params.n_ctx = std::stoi(argv[i]);
         }
+        else if (arg == "-gqa" || arg == "--gqa") {
+            if (++i >= argc) {
+                invalid_param = true;
+                break;
+            }
+            params.n_gqa = std::stoi(argv[i]);
+        }
```
Comment (slaren) [inline examples/server/server.cpp:734]: The rest of this file places the opening braces on a new line, let's use the same everywhere for consistency.

Comment (IgnacioFDM) [reply examples/server/server.cpp]: Fixed. Also changed a couple of lines nearby that also used K&R style.

File: examples/server/server.cpp [hunk 2]
```diff
@@ -608,6 +608,7 @@ static void server_print_usage(const char *argv0, const gpt_params &params,
     fprintf(stderr, "  -v, --verbose         verbose output (default: %s)\n", server_verbose ? "enabled" : "disabled");
     fprintf(stderr, "  -t N, --threads N     number of threads to use during computation (default: %d)\n", params.n_threads);
     fprintf(stderr, "  -c N, --ctx-size N    size of the prompt context (default: %d)\n", params.n_ctx);
+    fprintf(stdout, "  -gqa N, --gqa N       grouped-query attention factor (TEMP!!! use 8 for LLaMAv2 70B) (default: %d)\n", params.n_gqa);
```
Comment (digiwombat) [inline examples/server/server.cpp:611]: Considering #2338 that was recently merged, I say we go ahead and flip the help output to `stdout` across the board.

**Review by digiwombat — CHANGES_REQUESTED** [review]

File: examples/server/server.cpp (hunk 2 above)
Comment (IgnacioFDM) [reply examples/server/server.cpp:611]: Done

File: examples/server/server.cpp (hunk 1 above)
Comment (SlyEcho) [reply examples/server/server.cpp]: This will be changed in #2217 to the usual.

**Review by SlyEcho — APPROVED** [review]

**Pull request merged**